resources/conf/config.toml

```diff
@@ -30,6 +30,10 @@
   verifyHostName = true
   disableSslVerification = false
 
+[router.upstream.http2]
```

Comment:
Do we have to mention this in config.toml specifically? Since we only include frequently-changed or must-customize configs only by default.

Reply:
yes, these values are set by default in `default_config.go` and these are also mentioned in `config.toml.template`. removed them from `config.toml`.

Reply:
Then let's remove those from k8s configs as well

Reply:
@renuka-fernando removed